### PR TITLE
adding confirmation modal when enabling/disabing interfaces

### LINKED ISSUE #62: enabling or disable interfaces should ask for confirmation
Similar to https://github.com/kytos-ng/topology/issues/202 we should include the support for asking for confirmation when enabling/disabling interfaces

### LINKED ISSUE #94: enabling or disabling interfaces throw an error and does not change table view
While doing tests for issues https://github.com/kytos-ng/ui/issues/62 and https://github.com/kytos-ng/topology/issues/217 I noticed that when enabling / disabling interfaces, the Notification panel was not showing up and later investigations also showed that an error is threw on the console when calling that feature.

Screenshot 2024-10-07 at 10 25 18

### PR DESCRIPTION
**Pull request by italovalcy**
Closes #62 

Closes #94

Heads-up: this PR sits on top of PR #91 

### Summary

See updated changelog file.

### Local Tests

After applying this fix (along with https://github.com/kytos-ng/topology/pull/218):

Oct-07-2024 10-19-25

### End-to-End Tests

N/A

### CODE REVIEW

File: src/kytos/interfaceInfo.vue [hunk 1]
```diff
@@ -213,8 +226,7 @@ export default {
           icon: 'cog',
         }
         _this.next_state = _this.next_state == 'Enable'? 'Disable' : 'Enable'
-        _this.content['enabled'] = _this.next_state == 'Enable'? 'false' : 'true'
```
Comment (italovalcy) [inline src/kytos/interfaceInfo.vue:216]: Fix for issue #94 (vue3 does not allow the content props be changed like this)

**Review by viniarck — APPROVED** [review]
LGTM. Great fix. Nice to see `k-modal` being used for interfaces too. 

cc'ing @HeriLFIU and @rmotitsuki as reviewers

**Comment by HeriLFIU** [comment]
I tried the changes out myself and they worked great.

**Comment by HeriLFIU** [comment]
It's also a good use of the new modal component :)

**Review by HeriLFIU — APPROVED** [review]

**Pull request merged**